mmv1/products/vertexai/FeatureOnlineStore.yaml

```diff
@@ -41,65 +43,183 @@ description: Vertex AI Feature Online Store provides a centralized repository fo
 autogen_async: false
 examples:
   - !ruby/object:Provider::Terraform::Examples
-    name: vertex_ai_feature_online_store
-    primary_resource_id: feature_online_store
+    name: 'vertex_ai_featureonlinestore_with_bigtable'
+    primary_resource_id: 'featureonlinestore'
+    primary_resource_name: "fmt.Sprintf(\"terraform%s\",
+      context[\"random_suffix\"\
+      ])"
     vars:
-      name: example_feature_online_store
+      name: 'terraform2'
+      project: 'appeng-flex'
```

Comment:
I don't see a project being used in this example, or any of the examples actually. Do you need it here? Same question for both `test_env_vars`: `org_id` and `billing_account`.

Reply:
Ah, It was a miss as I assumed a test project would be provisioned to run the changes or similar, since this was present in other resources I've seen where they are not being utilized. This is now updated.